eva_submission/evapro/eload_metadata_loader.py

```diff
@@ -0,0 +1,29 @@
+import json
+import os
+
+from eva_submission.eload_submission import Eload
+from eva_submission.submission_config import EloadConfig
+
+
+class EloadMetadataJsonLoader(Eload):
+
+    def __init__(self, eload_number: int, config_object: EloadConfig = None):
+        super().__init__(eload_number, config_object)
+        self.metadata_json_path  = os.path.join(self._get_dir('ena'), 'metadata_json.json')
+        with open(self.metadata_json_path) as open_file:
+            self.metadata_json = json.load(open_file)
+
+    def get_experiment_types(self, analysis_accession):
+        analysis_alias_dict = self.eload_cfg.query('brokering','ena','ANALYSIS')
+        analysis_aliases = [a_alias for a_alias, a_accession in analysis_alias_dict.items() if a_accession == analysis_accession]
+        if len(analysis_aliases) != 1:
+            self.error(f'No experiment types can be found for {analysis_accession} accession')
+            return []
+        analysis_alias = self._unique_alias(analysis_aliases[0])
```

Comment:
Trying to remember how this works... this is needed because we're using the metadata json that we brokered to ENA rather than the one submitted, is that right?  But wouldn't the config `self.eload_cfg.query('brokering','ena','ANALYSIS')` also be using the unique alias?